### ISSUE TITLE
CombinedIngress Repo has no automated mirroring to GitLab

### ISSUE THREAD

**Issue by glogiotatidis** [mozmeao/infra#1305]
https://github.com/mozmeao/combinedIngress

was this a victim of the GitLab mirroring change? /cc @duallain

**Comment by duallain** [comment]
yeah, that would make sense.  I think we should move this repo to infra-services, since we have a docker step for wrangler.  Or, make this public, which would let us get mirroring working again.

**Comment by glogiotatidis** [comment]
I vote for consolidating this into `infra-services`
